### DISCUSSION TITLE
Getting basset error

### DISCUSSION THREAD

**Discussion by smargasy** [Laravel-Backpack/community-forum#824] [Q&A (Help)]
fixed

**Comment by pxpm** [comment]
Hey @smargasy happy it's fixed. 🙏 

You are totally free to do whatever you want with what you write, but I highly encourage you to leave your questions up, and if you come up with a solution, just write it as a comment and close the issue. 

Even dumb things, that happen to everyone, all the time helps future "us" that will bump in the same thing eventually someday 👍 

Cheers
